Vagrantfile

```diff
@@ -6,8 +6,8 @@ Vagrant.configure("2") do |config|
     config.vm.box = "minimal/trusty64"
 
     # Create forwarding ports for client-guest machine access via localhost.
-    config.vm.network :forwarded_port, guest: 80, host: 8080
-    config.vm.network :forwarded_port, guest: 6000, host: 6000
+    config.vm.network :forwarded_port, guest: 80, host: 8080, auto_correct: true
+    config.vm.network :forwarded_port, guest: 5000, host: 5000, auto_correct: true
```

Comment:
Not a standard port, and we might run into similar problems. Why not use something like 9000? Or a bit uncommon, 7777 or 7001.

Obviously, we would allocate this more robustly later on.